pyramid_openapi3/tests/test_validation.py

```diff
@@ -135,7 +135,7 @@ def test_view_raises_valid_http_exception(self) -> None:
         """
         from pyramid.httpexceptions import HTTPBadRequest
 
-        def view_func(*args):
+        def view_func(*args: t.Any) -> Exception:
```

Comment:
A bit fishy, since this is not return statement.

Reply:
Agreed, but couldn't come up with anything better.